client/src/components/form/DragAndDrop.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { RiDeleteBinLine, RiFileLine, RiInformation2Line } from '@remixicon/react';
+import { Callout, Divider, Select, SelectItem, TextInput } from '@tremor/react';
+import { useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+
+interface File {
+    path: string;
+    size: number;
+}
+
+function classNames(...classes: string[]) {
+    return classes.filter(Boolean).join(' ');
+}
+
+export function FileUpload() {
```

Comment:
It's not a very important problem and also it's not about your work, but about this component, but in the state we use dragging the files, if we drag one, we cannot add files to the previous and it will be replaced by the new one.

Reply:
you can drag more files, if you drag it together, maybe i add a comment in the user guidelines